### PR TITLE
Relationships

### PR DESCRIPTION
**Pull request by Lathrisk**
#90 
#35

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,6 +40,7 @@
     "eslint-config-prettier": "^6.11.0",
     "eslint-plugin-prettier": "^3.1.3",
     "file-loader": "^6.0.0",
+    "flag-icons": "^6.1.1",
```
Comment (odscjames) [inline package.json:43]: I'm curious why the files also need to be checked in if it's included as a package?

Comment (Lathrisk) [reply package.json:43]: We include the images as part of the build script and this seemed easier than pulling them from the npm module (which designed to be used in a way we're not currently using them). I actually only included the npm module as a reference of where we got the flag files from.

Comment (Lathrisk) [reply package.json:43]: I could possibly pull the images from the node_modules folder when we build, but I can't remember how stable the folder structure is. This also means the flags are explicitly committed as part of the tool.

Comment (odscjames) [inline package.json:43]: I note this is not locked and we talked about how you might want to lock it, but if you're happy that's fine.

**Review by odscjames — APPROVED** [review]

**Pull request merged**